### ISSUE TITLE
Add .gitignore file

### ISSUE THREAD

**Issue by alexkscott** [8BitFishy/Predator-Prey#1]
gitignore files stop you from uploading files that aren't needed/wanted in your file.

Python will generate a load of files that are only needed locally and is best not to upload.

[gitignore docs](https://git-scm.com/docs/gitignore)

- [ ] [Generate gitignore file](https://www.toptal.com/developers/gitignore) and add to repo
- [ ] Delete `__pycache__` and `.idea` files.

How to delete files in repo, https://stackoverflow.com/a/2047477

TLDR
```bash
git rm --cached <file to delete>
git commit -m "delete file"
```
